2-ui/1-document/10-size-and-scroll-window/article.md

```diff
@@ -16,33 +16,33 @@ For instance, this button shows the height of your window:
 <button onclick="alert(document.documentElement.clientHeight)">alert(document.documentElement.clientHeight)</button>
 ```
 
-````warn header="Not `window.innerWidth/Height`"
-Browsers also support properties `window.innerWidth/innerHeight`. They look like what we want. So why not to use them instead?
+````warn header="Not `window.innerWidth/innerHeight`"
+Browsers also support properties like `window.innerWidth/innerHeight`. They look like what we want, so why not to use them instead?
 
-If there exists a scrollbar, and it occupies some space, `clientWidth/clientHeight` provide the width/height without it (subtract it). In other words, they return width/height of the visible part of the document, available for the content.
+If there exists a scrollbar, and it occupies some space, `clientWidth/clientHeight` provide the width/height without it (subtract it). In other words, they return the width/height of the visible part of the document, available for the content.
 
-...And `window.innerWidth/innerHeight` include the scrollbar.
+`window.innerWidth/innerHeight` includes the scrollbar.
 
-If there's a scrollbar, and it occupies some space, then these two lines show different values:
+If there's a scrollbar, and it occupies some space, then these two lines return different values:
 ```js run
 alert( window.innerWidth ); // full window width
 alert( document.documentElement.clientWidth ); // window width minus the scrollbar
 ```
 
-In most cases we need the *available* window width: to draw or position something. That is: inside scrollbars if there are any. So we should use `documentElement.clientHeight/Width`.
+In most cases, we need the *available* window width—to draw or position something within scrollbars if there are any. So we should use `documentElement.clientHeight/clientWidth`.
 ````
 
 ```warn header="`DOCTYPE` is important"
-Please note: top-level geometry properties may work a little bit differently when there's no `<!DOCTYPE HTML>` in HTML. Odd things are possible.
+Please note: top-level geometry properties may work a little bit differently when there's no `<!DOCTYPE HTML>` in HTML.
```

Comment:
I like the chatty little addition. Would leave it in.

Reply:
Doesn't bother me the way it was, and the statement that "JavaScript can be quite finicky" seems to go beyond the scope of what is being discussed.

Reply:
```suggestion
Please note: top-level geometry properties may work a little bit differently when there's no `<!DOCTYPE HTML>` in HTML. Odd things are possible.
```